react/wbooks/src/app/screens/Home/styles.module.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../../scss/colors';
+
+@import '../../../scss/variables';
+
+.books-container {
+  flex-wrap: wrap;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
its better to center by `justify-items: center;`

Reply:
There is no change doing that since the component father of this component is root and root is not flex.
Kike suggested me to use the margin